### PR TITLE
chore: use Rust 1.63.0

### PR DESCRIPTION
**Pull request by mlafeldt**
https://github.com/rust-lang/rust/blob/1.63.0/RELEASES.md

~~**Note: Currently based on #15492, which is required for CI to pass.**~~

### CODE REVIEW

File: core/inspector.rs [hunk 1]
```diff
@@ -438,6 +438,7 @@ struct InspectorWakerInner {
   isolate_handle: v8::IsolateHandle,
 }
 
+// SAFETY: TODO
```
Comment (mlafeldt) [inline core/inspector.rs:441]: These are now required by https://github.com/denoland/deno/blob/ee2f4e745c33797ac9fd20571e77cef73ea585bf/.cargo/config.toml#L25

Not sure what a good safety warning would be for all the cases.

File: ext/ffi/lib.rs [hunk 2]
```diff
@@ -363,7 +367,7 @@ impl NativeValue {
       }
       NativeType::ISize => {
         let value = self.isize_value;
-        if value > MAX_SAFE_INTEGER || value < MIN_SAFE_INTEGER {
+        if !(MIN_SAFE_INTEGER..=MAX_SAFE_INTEGER).contains(&value) {
```
Comment (mlafeldt) [inline ext/ffi/lib.rs:370]: It's arguable if that's actually an improvement. 🤔

**Comment by mlafeldt** [comment]
Ready for review. 🙏

**Comment by bartlomieju** [comment]
@mlafeldt thanks, we got a few more big PRs to land before we will be merging other PRs. I will review it as soon as I can

**Comment by mlafeldt** [comment]
Rebased now that https://github.com/denoland/deno/pull/15492 is in.

**Review by bartlomieju — APPROVED** [review]
LGTM, thanks @mlafeldt

**Pull request merged**